src/memorypad.cpp

```diff
@@ -253,20 +253,20 @@ struct MemoryPadTrackpad : OpaqueWidget {
     }
   }
 
-  void draw(const DrawArgs& args) override {
+  void drawLayer(const DrawArgs& args, int layer) override {
+    if (layer != 1 || !module) return;
```

Comment:
Stylistically, I'd prefer to have braces if thats ok:
```
if (layer != 1 || !module) {
  return;
}
```

Reply:
Done